FileLog.mqh

```diff
@@ -95,10 +96,14 @@ CFileLog::CFileLog():m_level(WARNING),
 //|             Level of Log message                                 | 
 //| @var bool print                                                  |
 //|             print to console                                     |
+//| @var bool bAppPreExist                                           |
+//|             write in append mode if file pre exist               |
 //+------------------------------------------------------------------+
-CFileLog::CFileLog(string file_name="example.log",ENUM_LOG_LEVEL level=3,bool print=false)
+CFileLog::CFileLog(string file_name="example.log",ENUM_LOG_LEVEL level=3,bool print=false, bool bAppPreExist = false)
 {
+  this.iFileRead=bAppPreExist?FILE_READ:0;
```

Comment:
just a formatting issue, but it would be nice to have some whitespace to make this easier to read.
```suggestion
  this.iFileRead = bAppPreExist ? FILE_READ : 0;
```